ci/test/00_setup_env_native_valgrind.sh

```diff
@@ -5,11 +5,11 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 export LC_ALL=C.UTF-8
-
+export TEST_RUNNER_EXTRA="--exclude feature_block"
```

Comment:
If the `feature_abortnode` test is causing issues that are non-trivial to fix, you might exclude it for now:

```suggestion
export TEST_RUNNER_EXTRA="--exclude feature_block,feature_abortnode"
```